### DISCUSSION TITLE
这个应用对路由器性能有要求吗

### DISCUSSION THREAD

**Discussion by kuangru52** [messense/aliyundrive-webdav#373] [Q&A] [answered]
有个非常老旧的百米生活hq61，大概是单核535mhz，16ROM，64RAM，刷OpenWrt试了一下，打开图片都费劲，电影几乎不能看

**Comment by messense** [comment, marked as answer]
要处理 https 请求加解密，另外可以调整下 buffer 大小默认是 10MB 减少内存分配开销（不过可能会降低下载速度）。

要不就换个新的路由器。

**Reply by kuangru52** [reply]
有新的，就是看看这个垃圾能不能利用起来
